### PR TITLE
Fix readme to point to correct must gather image

### PR DESCRIPTION
**Pull request by manishym**
*
The image pointed to did not exist.
*

### CODE REVIEW

**Review by yati1998 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ that expands its capabilities to gather Openshift Container Storage for informat
 
 ### Usage
 ```sh
-oc adm must-gather --image=quay.io/rhceph-dev/odf4-odf-must-gather-rhel9:latest-4.13
+oc adm must-gather --image=registry.redhat.io/odf4/odf-must-gather-rhel9:v4.13
```
Comment (agarwal-mudit) [inline README.md:9]: This is just a README, shouldn't we just use example instead of actual image path else we have to change it after every release.

**Comment by black-dragon74** [comment]
@yati1998 We have a working CI/CD. Yayy!!

**Comment by yati1998** [comment]
Thanks @black-dragon74 for your contribution. Finally we have our first automatic merge done.

**Pull request merged**